src/main/resources/assets/depthmeter/lang/de_de.json

```
@@ -0,0 +1,6 @@
+{
+    "depthmeter.sea_level_below":"%s meter unter dem Meeresspiegel",
+    "depthmeter.sea_level_above":"%s meter über dem Meeresspiegel",
+    "item.depthmeter.depthmeter" :"Höhenmaß",
+    "itemGroup.depthmeter.main": "Höhenmaß"
```

Hi ! Thank you for your contribution !
I'm a little annoyed by the name change on the main, I think it should stay the same as in English, because people who install the mod see the name in English and not in German.

I've made this concesion in French too to keep the same name and do not confuse users of potential international servers.

```suggestion
    "itemGroup.depthmeter.main": "Depth Meter"
```